packages/arb-token-bridge-ui/src/hooks/useCompleteMultiChainTransactions.ts

```diff
@@ -306,11 +306,16 @@ export const useCompleteMultiChainTransactions = () => {
   }, [mapData])
 
   if (loading || error) {
-    return { data: { transactions: [], total: undefined }, loading, error }
+    return {
```

Comment:
I don't think it's related to this PR, but the transactions just keep on loading indefinitely even if I want to see them or not. 
(I have 100s of txs on test net so they just keep on appending)
Maybe we should optimize for user's screen presence here.

Reply:
Infinite scroll will be included in the next PR. Then we will only fetch 20-30 txs at a time.